src/webview/static/base.css

```diff
@@ -569,7 +581,8 @@ hr {
   border-top: 1px solid hsla(0, 0%, 50%, 0.5);
 }
 
-/* Our own "sorry, unsupported" message for widgets. */
+/* Our own "sorry, unsupported" message for widgets.
+   Also used for showing that a message was sent by a muted user. */
 .widget {
```

Comment:
I was going to comment that it's a fragile solution to reuse something called "widget" for a purpose unrelated to widgets.

The next commit renames this to `special-message`, which seems like a fine solution. That commit should just come before this one so there isn't a concerning intermediate state :slightly_smiling_face:

Reply:
Fixed.